### DISCUSSION TITLE
How can i deploy vitesse on Vercel

### DISCUSSION THREAD

**Discussion by JoshDev1205** [antfu/vitesse#258] [General]
I wanna know, if i need something to consider to deploy a vitesse project on vercel

**Comment by thuongtruong1009** [comment]
Hi @JoshDev1205 !
- You can completely deploy your web to Vercel or any deployment service site. Because this is simply a web app and they are configured by @antfu automatically.
- Similar to Vercel, you can also deploy to netlify.
- Hope it helps you!
- If can, you can mark it as an answer for another guy to know about this (set category to Q&A).
- Thank you!
